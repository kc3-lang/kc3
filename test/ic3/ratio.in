quote 1/1
1/1
quote 1/2
1/2
quote 1/3
1/3
quote 1/10
1/10
quote 1/100
1/100
quote 1/1000
1/1000
quote 2/3
2/3
quote 3/3
3/3
quote 10/3
10/3
quote 100/3
100/3
quote 1000/3
1000/3
quote -1/3
-1/3
quote -123456/3
-123456/3
quote - -1/3
- -1/3
quote - -123456/3
- -123456/3
quote 1/3 + 1/6
1/3 + 1/6
quote 1/3 - 1/6
1/3 - 1/6
quote 2/3 * 5/6
2/3 * 5/6
quote 2/3 / 5/6
2/3 / 5/6
