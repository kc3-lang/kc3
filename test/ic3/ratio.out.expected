1/1
1/1
1/2
1/2
1/3
1/3
1/10
1/10
1/100
1/100
1/1000
1/1000
2/3
2/3
3/3
3/3
10/3
10/3
100/3
100/3
1000/3
1000/3
-1/3
-1/3
-123456/3
-123456/3
- -1/3
1/3
- -123456/3
123456/3
